lib/ansible/plugins/connection/iocage.py

```diff
@@ -70,6 +70,13 @@ def get_jail_uuid(self):
                              stderr=subprocess.STDOUT)
 
         stdout, stderr = p.communicate()
+
+        if stdout is not None:
+            stdout = stdout.decode('utf-8')
+
+        if stderr is not None:
+            stderr = stderr.decode('utf-8')
```

Comment:
Code should utilize `ansible.module_utils._text.to_native` here instead.

Reply:
Done!